code/ui/manager/src/components/preview/FramesRenderer.tsx

```diff
@@ -51,6 +51,8 @@ export const FramesRenderer: FC<FramesRendererProps> = ({
   storyId = '*',
 }) => {
   const version = refs[refId]?.version;
+  const storyIdRef = useRef(storyId);
+  const viewModeRef = useRef(viewMode);
```

Comment:
Can you explain why these refs are needed?

Reply:
The idea is to lock the values, so they won't change, thus won't trigger updates.
But I'll refactor this to have the same effect.

Reply:
@ndelangen as long as you don't depend on them in the dependencies array to `useEffect`, it doesn't matter.